style.css

```diff
@@ -0,0 +1,65 @@
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  width: 375px;
+  margin: 0 37%;
+}
+
+.header {
+  height: 92px;
+  width: 355px;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding: 10px;
+  color: white;
+}
+
+.headline {
+  width: 375px;
+  height: 590px;
+  background-color: #1c1a19;
+}
```

Comment:
- [ ] Consider using CSS background-image for the graphic(illustration) as it is not part of the content.
- It should look like this:

![image](https://user-images.githubusercontent.com/52143815/191854899-4b5741ab-1745-46ff-ad26-7bbb7c6d70a1.png)